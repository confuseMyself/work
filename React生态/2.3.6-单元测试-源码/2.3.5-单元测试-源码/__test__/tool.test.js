import sum from '../components/add'
import {
  reverse
} from "../components/add";


function arr() {
  return [1,2,3]
}

test('adds 1 + 2 to equal 3', () => {
  expect(sum(1, 2)).toBe(3);
});


test('string reverse', () => {
	expect(reverse('abc')).toBe('cba');
})

test('test contain', () => {
  expect(arr()).toMatchSnapshot(3);
});
