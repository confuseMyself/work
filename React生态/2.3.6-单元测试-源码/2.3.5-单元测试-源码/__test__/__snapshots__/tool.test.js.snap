// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test contain 1`] = `
Array [
  1,
  2,
  3,
]
`;
